user-guide/query-examples-hacks/redash-hacks.md

```diff
@@ -0,0 +1,64 @@
+# Redash Hacks
+
+We recommend to add your favorite hacks as snippets for quick and easy access when writing your queries.
+Learn how to create snippets [here](../queries/writing_queries.md#query_snippets)
+
+Documented hacks up-to-date are:
+* [Conditional Formatting](#conditional-formatting)
+* [Clickable URLs in tables](#clickable-urls-in-table)
+* [Images Inside tables](#images-inside-table)
+* [Default Parameter Value](#default-param-value)
+
+## Conditional Formatting and General Text Formatting {#conditional-formatting}
+
+While Redash doesn't naturally support conditional formatting, this can be bypassed quite easily with some html tags.
+
+```CASE
```

Comment:
I cappd the keywords and changed ``` to ```sql but the atom preview looks weird, I hope it's just another glitch in atom and it was saved ok